src/Weather.js

```diff
@@ -0,0 +1,12 @@
+class Weather {
+    stormy = false;
```

Comment:
I don't think I needed a variable here, I could simply return the boolean.
I would make this a static method in future, which could be passed airport location to assess the weather